server/home/home-common/src/main/java/io/holoinsight/server/home/common/model/TaskEnum.java

```diff
@@ -23,21 +23,13 @@ public enum TaskEnum {
 
   TENANT_INTEGRATION_GENERATED("TENANT_INTEGRATION_GENERATED", "TENANT_INTEGRATION_GENERATED"),
 
-  TENANT_OB_SERVER_SYNC("TENANT_OB_SERVER_SYNC", "TENANT_OB_SERVER_SYNC"), TENANT_CLOUD_MODEL_SYNC(
-      "TENANT_CLOUD_MODEL_SYNC", "TENANT_CLOUD_MODEL_SYNC"),
+  TENANT_OB_SERVER_SYNC("TENANT_OB_SERVER_SYNC", "TENANT_OB_SERVER_SYNC"),
 
-  CLEAN_ALERT_HISTORY("CLEAN_ALERT_HISTORY", "CLEAN_ALERT_HISTORY"),
+  TENANT_CLOUD_MODEL_SYNC("TENANT_CLOUD_MODEL_SYNC", "TENANT_CLOUD_MODEL_SYNC"),
```

Comment:
Migrate from opensource to enterprise
TENANT_OB_SERVER_SYNC
TENANT_CLOUD_MODEL_SYNC